DOCKER-RELAYER.md

```diff
@@ -0,0 +1,353 @@
+- `ENABLE_STARKNET`: Enable Starknet protocol (default: `false`)
+- `ENABLE_ICP`: Enable ICP protocol (default: `false`)  
+- `ENABLE_ETHEREUM`: Enable Ethereum protocol (default: `false`)
+
+#### Near Protocol
+- `NEAR_NETWORK`: Network name (default: `testnet`)
+- `NEAR_RPC_URL`: RPC endpoint (default: `https://rpc.testnet.near.org`)
+- `NEAR_CONTRACT_ID`: Contract address (default: `calimero-context-config.testnet`)
+- `NEAR_ACCOUNT_ID`: Account ID (custom credentials)
+- `NEAR_PUBLIC_KEY`: Public key (custom credentials)
+- `NEAR_SECRET_KEY`: Secret key (custom credentials)
+- `NEAR_DEFAULT_SECRET_KEY`: Default secret key for testnet
+
+#### Starknet Protocol
+- `STARKNET_NETWORK`: Network name (default: `sepolia`)
+- `STARKNET_RPC_URL`: RPC endpoint
+- `STARKNET_CONTRACT_ID`: Contract address
+- `STARKNET_ACCOUNT_ID`: Account ID
+- `STARKNET_PUBLIC_KEY`: Public key
+- `STARKNET_SECRET_KEY`: Secret key
+- `STARKNET_DEFAULT_SECRET_KEY`: Default secret key
+
+#### ICP Protocol
+- `ICP_NETWORK`: Network name (default: `local`)
+- `ICP_RPC_URL`: RPC endpoint (default: `http://host.docker.internal:4943`)
+- `ICP_CONTRACT_ID`: Contract ID
+- `ICP_ACCOUNT_ID`: Principal ID
+- `ICP_PUBLIC_KEY`: Public key
+- `ICP_SECRET_KEY`: Secret key
+- `ICP_DEFAULT_SECRET_KEY`: Default secret key
+
+#### Ethereum Protocol
+- `ETHEREUM_NETWORK`: Network name (default: `sepolia`)
+- `ETHEREUM_RPC_URL`: RPC endpoint
+- `ETHEREUM_CONTRACT_ID`: Contract address
+- `ETHEREUM_ACCOUNT_ID`: Account address
+- `ETHEREUM_SECRET_KEY`: Private key
+- `ETHEREUM_DEFAULT_SECRET_KEY`: Default private key
+
+## Deployment Scenarios
+
+### 1. Production Deployment
+
+```yaml
+# docker-compose.prod.yml
+version: '3.8'
+services:
+  relayer:
+    image: calimero-relayer:latest
+    ports:
+      - "63529:63529"
+    environment:
+      - ENABLE_NEAR=true
+      - NEAR_SECRET_KEY=${NEAR_PROD_SECRET_KEY}
+      - ENABLE_ETHEREUM=true  
+      - ETHEREUM_SECRET_KEY=${ETHEREUM_PROD_SECRET_KEY}
+    restart: unless-stopped
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:63529/health"]
+      interval: 30s
+      timeout: 5s
+      retries: 3
+```
+
+```bash
+# Deploy to production
+export NEAR_PROD_SECRET_KEY="ed25519:..."
+export ETHEREUM_PROD_SECRET_KEY="0x..."
+docker-compose -f docker-compose.prod.yml up -d
+```
+
+### 2. Development Environment
+
+```bash
+# Start development environment with local networks
+./scripts/relayer/start-dev.sh
+
+# This will start:
+# - ICP dfx local network
+# - Ethereum Anvil devnet  
+# - Relayer with development credentials
+```
+
+### 3. Testing Environment
+
+```bash
+# Start with testnet credentials
+export NEAR_DEFAULT_SECRET_KEY="ed25519:3D4YudUQRE39Lc4JHghuB5WM8kbgDDa34mnrEP5DdTApVH81af7e2dWgNPEaiQfdJnZq1CNPp5im4Rg5b2rKtXFv"
+export STARKNET_DEFAULT_SECRET_KEY="0x0178eb2a625c0a8d85b0a5fd69fc879f9884f5205ad9d1ba41db0d7d1a77950a"
+
+docker-compose -f docker-compose.relayer.yml up -d
+```
```

Comment:
Please remove secrets (even if they are randomly generated) from here, as well as from the PR description.